Add new parameter: repoVisibility for the action: publish:gitea (#24383)
backstage/backstage: plugins/scaffolder-backend-module-gitea/README.md

      password: '<GITEA_LOCALHOST_PASSWORD>'
```

**Important**: As backstage will issue `HTTPS/TLS` requests to the gitea instance, it is needed to configure `gitea` with a valid certificate or at least with a

> self-signed certificate `gitea cert --host localhost -ca` trusted by a CA authority. Don't forget to set the env var `NODE_EXTRA_CA_CERTS` to point to the CA file before launching backstage or to ignore using the following Node.js parameter: `NODE_TLS_REJECT_UNAUTHORIZED=0` !

Reviewer: ```suggestion
self-signed certificate `gitea cert --host localhost -ca` trusted by a CA authority. Don't forget to set the env var `NODE_EXTRA_CA_CERTS` to point to the CA file before launching backstage or you can set temporarily `NODE_TLS_REJECT_UNAUTHORIZED=0` but this is not recommended for production!
```

When done, you can create a template which:

- Declare the `RepoUrlPicker` within the `spec/parameters` section to select the gitea host and to provide the name of the repository

> - Add an enum list allowing the user to define the visibility about the repository to be created: `public` or `private`. If this field is omitted, `public` is then used by the action.

Author: Do you mean that part of this README, the syntax to be used should be: `'public'`, `'private'` AND not `public`, `public` ?

> - Add an enum list allowing the user to define the visibility about the repository to be created: `public` or `private`. If this field is omitted, `public` is then used by the action.

Reviewer 2: like this 😄

```suggestion
- Add an `enum` list allowing the user to define the visibility about the repository to be created: `public` or `private`. If this field is omitted, `public` is then used by the action.
```

> - Add an enum list allowing the user to define the visibility about the repository to be created: `public` or `private`. If this field is omitted, `public` is then used by the action.

Author: I highlighted the enum word: 9c65898
I dont have vale errors locally
```
vale --config .vale.ini /Users/cmoullia/code/janus-idp/fork-backstage/plugins/scaffolder-backend-module-gitea/README.md
✔ 0 errors, 0 warnings and 0 suggestions in 1 file.
```
